### DISCUSSION TITLE
netbox  ipam issues

### DISCUSSION THREAD

**Discussion by lasag-2511** [netbox-community/netbox#13395] [General]
Hi all,
Is there anyway that we could get DNS name automatically with respect to IP address, while scanning the IP in the network.

**Comment by mtinberg** [comment]
This is something that isn't in the Netbox core experience and is unlikely to be added, but there are some DNS configuration management plugins to Netbox for managing and templating DNS zone files, and you can write your own tool using Ansible or another scripting language over the REST API to provide your own IP reconciliation and import of relevant data into Netbox if you want.


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: lasag-2511 ***@***.***>
Sent: Monday, August 7, 2023 3:28 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] netbox add DNS Name under ipam (Discussion #13395)


Hi all,
Is there anyway that we could get DNS name automatically with respect to IP address, while scanning the IP in the network.

**Comment by candlerb** [comment]
> while scanning the IP in the network

Netbox doesn't scan your network, so the functionality to lookup DNS names and insert them in the (Netbox) IPAddress records would need to be done as part of whatever scanning tool you're using.

**Comment by lasag-2511** [comment]


**Reply by candlerb** [reply]
This sounds like it should be a separate thread/discussion.  You'll need to provide details of the symptoms (e.g. what you actually see when you try to access netbox), and what checks you've already done (e.g. `systemctl status netbox`, `journalctl -eu netbox`)

**Comment by lasag-2511** [comment]
Hi python netbox module for creating ip address doesnot work anymore.

netbox.ipam.create_ip_address
